README.md

```diff
@@ -2,3 +2,10 @@
 -------------------
 
 Examples for cpplinq
+
+## Build Status
+|Travis|AppVeyor|
+|------|--------|
+|[![Build Status](https://travis-ci.org/JorTurFer/cpplinq_examples.svg?branch=master)](https://travis-ci.org/JorTurFer/cpplinq_examples)|[![Build status](https://ci.appveyor.com/api/projects/status/lgo0245i3qg3duo0?svg=true)](https://ci.appveyor.com/project/kabestrus/cpplinq-examples)|
```

Comment:
You should change this badges to your own badges

Reply:
Great! 👍